### PR TITLE
Repo init

### PR DESCRIPTION
**Pull request by vkaracic**

### CODE REVIEW

**Comment by vkaracic** [comment]
Note: `VInput` and user login is not implemented properly yet, it is still in TODO.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "edifyr",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon -r dotenv/config ./server/bin/www",
+    "migrate": "node_modules/.bin/sequelize db:migrate",
+    "seed": "node_modules/.bin/sequelize db:seed:all",
+    "seed:undo": "node_modules/.bin/sequelize db:seed:undo",
```
Comment (marjev) [inline package.json]: You're missing `nodemon -r dotenv/config` for `migrate`, `seed` and `seed:undo`. Without it, .env settings won't be loaded and I end up with default values.

Comment (marjev) [reply package.json]: Nice addition would be to add an alias, suggestion: `db`.
`"db": "node -r dotenv/config node_modules/.bin/sequelize"`
That way, you could call `npm run db migrate` and remove duplicate statements from `migrate`, `seed` and `seed:undo`.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+.vuepress/dist
+
+# Serverless directories
+.serverless/
+
+# FuseBox cache
+.fusebox/
+
+# DynamoDB Local files
+.dynamodb/
```
Comment (marjev) [inline .gitignore]: Add .vscode/

**Comment by marjev** [comment]
Consider using `poi` for running client side. We're using it on most (if not all of our projects).
One of the reasons is because it's zero-config so it should be almost effortless to implement it.

**Pull request closed without merging**